src/platform_impl/windows/window_state.rs

```diff
@@ -219,6 +221,9 @@ impl WindowFlags {
         if self.contains(WindowFlags::MAXIMIZED) {
             style |= WS_MAXIMIZE;
         }
+        if self.contains(WindowFlags::IGNORE_CURSOR_EVENT) {
+            style_ex |= WS_EX_TRANSPARENT | WS_EX_LAYERED;
```

Comment:
In #1815, we decided to remove the `WS_EX_LAYERED` style as it is documented as being incompatible with the CS_OWNDC window class style.
When I test stuff (not this PR, mind you) on my own machine it seems to work fine, but the documentation nevertheless says it is incompatible.

Reply:
Good point.. maybe we can get rid of the `CS_OWNDC` part as it's mainly/only for OpenGL and I believe we can get around it. Will play around with this one a bit in the next days.

--------------

Would be nice to have reference link added here regarding hit-test behavior (https://docs.microsoft.com/de-de/windows/win32/winmsg/window-features?redirectedfrom=MSDN#layered-windows)